### PR TITLE
[Optimize][Web]optimize_web_catalog

### PR DESCRIPTION
**Pull request by zackyoungh**
## Verify this pull request
chrome_nlAF9mxxC7
chrome_apt4rP6usa
chrome_kPDPHpGXcc

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by Zzm0809 — APPROVED** [review]
LGTM

File: dinky-executor/src/main/java/org/dinky/executor/VariableManager.java [hunk 1]
```diff
@@ -53,6 +53,7 @@
  * @since 2021/6/7 22:06
  */
 public final class VariableManager {
+    public static final VariableManager INSTANCE = new VariableManager();
```
Comment (aiwenmo) [inline dinky-executor/src/main/java/org/dinky/executor/VariableManager.java]: This can cause variable conflicts between different tasks.

**Review by aiwenmo — CHANGES_REQUESTED** [review]

**Comment by Zzm0809** [comment]
@aiwenmo PTAL

**Review by aiwenmo — APPROVED** [review]
LGTM

**Pull request merged**